src/viking.js

```diff
@@ -1,11 +1,72 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+        this.health=health
+        this.strength=strength
+    }
+    attack(){
+        return this.strength
+    }
+    receiveDamage(damage){
+        this.health-= damage
+
+    }
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier{
+    constructor(name,health, strength){
+        super(health, strength)
+        this.name=name
+    }
+    receiveDamage(damage){
+        this.health-= damage
+        if(this.health>0){
+            return `${this.name} has received ${damage} points of damage`
+        }else {
```

Comment:
The same as the previous, but also to keep the pattern, as you added a space after the `else`
```suggestion
        } else {
```